pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl

```diff
@@ -0,0 +1,116 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/TrimAdapters.wdl" as TrimAdapters
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/StarAlign.wdl" as StarAlignFastq
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/Picard.wdl" as Picard
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/FeatureCounts.wdl" as CountAlignments
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/LoomUtils.wdl" as LoomUtils
+
+
+workflow SmartSeq2SingleNucleus {
+  meta {
+    description: "Process SmartSeq2 snRNA-Seq data, include reads alignment, QC metrics collection, and gene expression quantitication"
+  }
+
+  input {
+    # load annotation
+    File genome_ref_fasta
+    # load index
+    # ref index name
+
+    File tar_star_reference
+    File annotations_gtf
+
+    # samples
+    String input_id
+
+    String? input_name
+    String? input_id_metadata_field
+    String? input_name_metadata_field
+
+    String output_name
+
+    File adapter_list
+    File fastq1
+    File? fastq2
+    Boolean paired_end
```

Comment:
Didn't we say that we weren't supporting single end in the first version of this pipeline? What happens if a user puts in `false` for this pipeline?

Reply:
We removed paired_end input.